NitroxClient/GameLogic/PlayerManager.cs

```diff
@@ -75,12 +77,22 @@ public void RemovePlayer(ushort playerId)
             {
                 opPlayer.Get().Destroy();
                 playersById.Remove(playerId);
+                DiscordController.Main.UpdateDRPDiving(GetPlayerCount() + 1);
             }
         }
 
         private GameObject CloneLocalPlayerBodyPrototype()
         {
             return Object.Instantiate(localPlayer.BodyPrototype);
-        }        
+        }
+
+        public int GetPlayerCount()
+        {
+            if (playersById == null)
```

Comment:
Do we have to check for null here?  It looks like the object is always initialized above.